docs/Splashkit/Applications/Arcade Machines/Guide to Contribute a Game/Contributors Guide.md

```diff
@@ -0,0 +1,152 @@
+# Guide to Contribute a Game to the Arcade Machine 
+
+### This guide takes you through the steps required for your game to be added into the arcade-machine library.  
+---
+<br>
+Table of contents
+   
+1. 
+2. [Coding](#coding)
+   - [Quit Request](#quit-request)
+   - [Window Size](#window-size)
+   - [Window Border](#window-border)
+3. [Controls]()
+4. [Compiling]()
+5. [Artwork]()
+6. [Configuration]()
+7. [Content]()
+8. [Contributing]()
+
+---
+
+<br>
+
+## **Coding**
+To make the game accessible and controllable by the arcade machine, some additional code or changes are required
+
+<br>
+
+### *Quit Request*
+Your game must be able to be exited using the escape key
+This can be achieved by including the following command in your main loop. 
+
+```cpp
+int main()
+{
+    while(!key_down(ESCAPE_KEY))
+    {
+        // game play
+    }
+}
+```
+<br>
+
+### *Window Size*
+The window size of your game cannot exceed 1600 x 900, this is to allow your game to sit neatly inside the arcade-machine itself. Similarly, but mainly for aesthetic purposes, a minimum window size of 640 x 480 is expected. 
```

Comment:
I would specify which is width and height (being explicitly clear to anyone reading), and have simplified the text in the suggestion

```suggestion
The window size of your game cannot exceed 1600 x 900, this is to allow your game to sit neatly inside the arcade-machine itself. Similarly, there is a minimum window size of 640 x 480, to ensure visibility for the user.
```